src/components/common/FooterBtn/FooterBtn.Style.ts

```diff
@@ -0,0 +1,42 @@
+import { css } from "@emotion/react";
+
+import { theme } from "@/styles/theme";
+
+export const footerBtnStyle = (
+	transparent?: boolean,
+	isSquare?: boolean,
+	isPositionStatic?: boolean,
+) => {
+	return css`
+		display: flex;
+		align-items: center;
+		justify-content: center;
+		width: 100%;
+		height: ${isSquare ? "3.44rem" : "5.5rem"};
+		padding: ${!isSquare && "0 1.5rem"};
```

Comment:
height가 버튼 배경 영역 때문에 잡은것 맞을까요? 
맞다면 이 부분을 padding으로만 할 수 있을것 같은데 어떤가요? 
아래처럼 사각형이 아닐때만 사방에 패딩을 주면 height가 없어도 동일하더라구요 
```suggestion
		padding: ${!isSquare && "1rem 1.5rem"};
```

Reply:
padding만 주는 방법도 있군여
수정했습니다.